test/route.js

```diff
@@ -681,6 +682,27 @@ describe('Router', function () {
             .expect(200, { user: 'tj', op: 'edit' }, done)
         })
 
+        it('should allow matching literal parenthesis within a group', function (done) {
+          var cb = after(3, done)
+          var router = new Router()
+          var route = router.route('/:user([a-z\\(\\)]+)')
```

Comment:
The PR should include a note in the history entry that these (and whatever other chars) now need to be escaped so folks know.

Reply:
Shouldn't that be created when releasing?

Reply:
No, because things get lost. The history file is updated with every commit as part of the commit itself.

Reply:
Ok. Which heading should I use?

Reply:
It doesn't matter all too much, as long as the release tooling (under `scripts/`) catches it. I would probably just do `2.x` as that is what the other commits in the 2.0 branch used.

Reply:
Done. Added everything I could find to `HISTORY.md`.